ess/src/API/EMBC.ESS/Managers/Admin/Mappings.cs

```diff
@@ -43,6 +43,7 @@ public MappingProfile()
                 ;
 
             CreateMap<Shared.Contracts.Suppliers.Supplier, Resources.Suppliers.Supplier>()
+                .ForMember(d => d.IsPrimarySupplier, opts => opts.Ignore())
```

Comment:
can this property be removed from the resource contract, or is it used upstream in the manager?